src/common/legacy_config_opts.h

```diff
@@ -1557,6 +1557,7 @@ OPTION(rgw_d3n_l1_datacache_size, OPT_LONGLONG)
 OPTION(rgw_d3n_l1_evict_cache_on_start, OPT_BOOL)
 OPTION(rgw_d3n_l1_libaio_read, OPT_BOOL)
 OPTION(rgw_d3n_l1_fadvise, OPT_INT)
+OPTION(rgw_d3n_l1_eviction_policy, OPT_STR)
 OPTION(rgw_d3n_l2_distributed_datacache_enabled, OPT_BOOL)
```

Comment:
should we re-squash this so (e.g.) this pr doesn't contain l2 commits?

Reply:
sure, will squash the L2 removal commit